### ISSUE TITLE
Parameters: [ScalingMinSize, ScalingMaxSize] do not exist in the template

### ISSUE THREAD

**Issue by nicholasgriffintn** [stelligent/mu#401]
hey,

whenever I configure an aurora serverless with a min or max size i get this error:

`Parameters: [ScalingMinSize, ScalingMaxSize] do not exist in the template`

how do I configure the min . or max size for aurora if not that? because the default 256 is still coming through, which i definitely don't want.

here's the config:

```
  database:
      name: wordpress
      engine: aurora
      instanceClass: db.t2.small
      iamAuthentication: false
      masterUsername: admin
      engineMode: serverless
      allocatedStorage: 10

      environmentConfig:                        
        acceptance:
          engineMode: serverless
          minSize: 2
          maxSize: 2
        production:
          engineMode: serverless
          minSize: 2
          maxSize: 8

```

**Comment by mneil** [comment]
Looks like there's an issue with the expected parameter name in the code. I've opened a PR to get it fixed. It'll should be available next release.

**Comment by cplee** [comment]
fixed in #402

**Comment by nicholasgriffintn** [comment]
Thanks for the quick fix will update to that later.
